### PR TITLE
Reducing the number of broken links in posts

### PR DESCRIPTION
**Pull request by edlongman**
Having looked at the broken links I have made a decision as to what to do with them.

Some have been removed if that is the best course of action

Some have been redirected to Wikipedia for game specific info for that year.

### CODE REVIEW

File: _posts/2014-10-26-student-robotics-2015-starts-with-a-range-of-new-kit.md [hunk 1]
```diff
@@ -18,10 +18,10 @@ The game: Capture the Flag
 
 <img class="right" src="{{ site.baseurl }}/images/content/arena-2015.png" title="The 2015 arena layout" alt="An image of the 2015 arena layout">
 
-This year's competition is all about retrieving flags from around the arena and putting them into the competitor's zone 
+This year's competition is all about retrieving flags from around the arena and putting them into the competitor's zone
 to get points. The flags are built from wooden cubes on ball casters.
 
-Read more about the [new game](/schools/game) or for full details please reference the [rulebook](/resources/2015/rulebook.pdf).
+Read more about the [new game](https://en.wikipedia.org/wiki/Student_Robotics#2015:_.22Capture_the_Flag.22) or for full details please reference the rulebook (now removed)
```
Comment (richardbarlow) [inline _posts/2014-10-26-student-robotics-2015-starts-with-a-range-of-new-kit.md]: The rulebooks from previous years are not currently hosted anywhere. However, I think that we should probably add them back in either in this repo, or more likely the docs repo. They provide an interesting insight into the history of the games.

**Comment by edlongman** [comment]
Maybe an archive section? Just where resources could be stored.

**Comment by PeterJCLaw** [comment]
There's an outstanding ticket (#29; note the difference from #3 which
appears to be more about historic competition results) to add back an
archive page about the old games.

We should _not_ be linking to our own wikipedia page for this sort of
information (doing so looks terrible [worse than a dead link IMO] and
will likely cause wikipedia to remove the content).

**Comment by edlongman** [comment]
Would it be useful to put the rulebooks for the years where there is missing info into the resources, #29  Possibly then the game pages as detailed in #3 could be built upon that.

**Comment by PeterJCLaw** [comment]
Thanks for this, however many of these have since been fixed. Some remain, however it seems most likely that a fresh approach is going to be more efficient than trying to merge this PR given it's age.

**Pull request closed without merging**